frontend/components/ActivityPool.vue

```diff
@@ -91,12 +105,101 @@ function showInfo(id: string) {
         }
     });
 }
+
+const confirmDelete = (event: Event) => {
+    confirm.require({
+        target: event.currentTarget as HTMLElement,
+        header: t.value("activity.delete.header"),
+        message: t.value("activity.delete.confirm"),
+        icon: "pi pi-exclamation-triangle",
+        rejectClass: "hover:underline",
+        acceptClass:
+            "text-error dark:text-error-dark hover:underline font-bold",
+        rejectLabel: t.value("common.button.cancel"),
+        acceptLabel: t.value("journey.delete"),
+        accept: () => {
+            toast.add({
+                severity: "info",
+                summary: t.value("common.toast.info.heading"),
+                detail: t.value("delete.journey.toast.message"),
+                life: 3000,
+            });
+            deleteActivity();
+        },
+    });
+};
+
+async function deleteActivity() {
```

Comment:
Ich weiß nicht wo das genau hingehört aber das Problem ist dieses. WEnn man auf Bearbeiten -> Löschen und dann auf abbrechen geht ist der background nicht mehr fixed
davor: 
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/07feb05e-6649-49ab-b8bc-03d240f9973f)
Danach (du kannst es erkennen weil da eine scrollbar kommt)
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/2198ffdc-bf16-4a98-825a-28faa5bae693)

Reply:
not really fixable